bitnami/spark/values.yaml

```diff
@@ -286,6 +286,11 @@ master:
   ## @param master.resources.limits The resources limits for the container
   ## @param master.resources.requests The requested resources for the container
   ##
+  ## NFS
+  ## nfs_server_ip: ""   # The IP of NFS Server
+  ## nfs_server_path: "" # The NFS server volume path
+  ## nfs_mount_path: ""  # The local path that will mount the NFS share volume
+
```

Comment:
Use the same approach as the rest of the file:

* Uncomment the parameters.
* Add a comment like this `@param master.nfs_server_ip` for each of them.
* Run the [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) after that.
Apart from that, don't place it in the middle of a section. For example, these lines are in the middle of `Container resource requests and limits`